models/jira__timestamp_issue_field_history.sql

```diff
@@ -104,6 +115,10 @@ final as (
         left join issue_types
             on cast(issue_types.issue_type_id as {{ dbt.type_string() }}) = create_validity_periods.issue_type
 
+        {% elif col.name|lower == 'team' and var('jira_using_teams', True) %}
+        left join teams
+            on cast(teams.team_name as {{ dbt.type_string() }}) = create_validity_periods.team
```

Comment:
with the suggested reversion in `int_jira__issue_field_history`, the team value is still in the id format, so we will want to make the following revision:


```suggestion
            on cast(teams.team_id as {{ dbt.type_string() }}) = create_validity_periods.team
```